RIT-MDRC/Catbot
# Feat/cli v2

# Feature
* catbot cli v2
* postinstaller 
   - Creates a globally runnable folder that can be executed by any user
   - This is beneficial because only the devs needs to run the install command and the other users will automatically have up to date code
* catbot start -D command
   - because catbot install command now runs a global copy of the sec folder it makes it hard to run the repo’s src folder
   - This is where the -D parameter comes in. This argument will make the catbot start command run in the repo’s dev folder instead of global

# How to Test
* ssh into catbot
* cd into the repo
* run `sudo npm install -g`
* run `su - catbotuser`
* run `catbot start`
* Check that the up to date script is written

closes #17 
closes #14
